src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
+        Random random = new Random();
```

Comment:
move random to the class variables, like in: 
![image](https://user-images.githubusercontent.com/12864128/222900521-d01d27a8-23d3-4633-af65-0910802aef2f.png)

Reply:
comment not fixed